.github/actions/build-electron/action.yml

```diff
@@ -8,7 +8,7 @@ inputs:
     description: 'Target platform'
     required: true
   artifact-platform:
-    description: 'Artifact platform, should be linux, darwin or mas'
+    description: 'Artifact platform, should be linux, windows, darwin or mas'
```

Comment:
Can we change `windows` to `win32` to be consistent with node's `process.platform`?

I plan to rely on this for downloading `generated_artifacts_win32_x64` in https://github.com/electron/build-tools/pull/679. It's easy to change though so let me know!

Reply:
@samuelmaddock sure - we can make the artifact use `win32` instead of `windows`